front/pages/api/w/[wId]/data_sources/managed.ts

```diff
@@ -104,6 +94,39 @@ async function handler(
           },
         });
       }
+      switch (provider) {
```

Comment:
Let's move it just after the case POST for maximal sec clarity

Reply:
provider is validated only here @spolu.

Reply:
Then let's add a isBuilder gard before the method switch :+1: on top of this :pray:

Reply:
@spolu done!